docs/quick-start-guide.md

```
@@ -0,0 +1,117 @@
+```
+
+2. Make your changes and ensure tests pass:
+
+```bash
+make test
+make verify
+```
+
+3. Commit with a descriptive message following [conventional commits](https://www.conventionalcommits.org/):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)